spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -52,6 +52,9 @@
       before do
         stub_sign_in user
         session[:sign_in_flow] = sign_in_flow
+        session[:sign_in_page_visited_at] = Time.zone.now.to_s
+        freeze_time
```

Comment:
Does this automatically unfreeze after the specs are run, or does this need to be torn down to avoid affecting other specs being run?

Elsewhere we use this pattern:

```rb
around do |example|
  freeze_time { example.run }
end
```

Reply:
I see. Interesting how that works. I updated that controller.